packages/atst/README.md

```diff
@@ -0,0 +1,25 @@
+# @eth-optimism/atst (WIP)
+
+WIP sdk and cli tool for the attestation station. Currently in design phase. Any code here is purely experimental proof of concepts.
+
+## Dev setup
+
+To be able to run the cli commands with npx you must first link the node module so npm treats it like an installed package
```

Comment:
Should we link to the [npm documentation about the npm link](https://docs.npmjs.com/cli/v8/commands/npm-link) command?